### ISSUE TITLE
Resource Limit Exceeded Error

### ISSUE THREAD

**Issue by cruise-insights** [Microsoft/PowerBI-JavaScript#144]
Hi,
@ali-hamud - All my visuals in Power BI embedded Report are loading, except 2. Please help resolving it
Issues :

1)
![image](https://cloud.githubusercontent.com/assets/7963522/22874427/f02d2d6c-f1eb-11e6-8c42-97de8a3a2625.png)

2)
![image](https://cloud.githubusercontent.com/assets/7963522/22874379/adcd2418-f1eb-11e6-9ba7-e3edc07a27d3.png)

Thanks,

**Comment by ali-hamud** [comment]
Do you have a visual with a lot of data points?

**Comment by cruise-insights** [comment]
@ali-hamud 
Yes, the report uses direct query and a Data Factory Pipeline. I can't sample dataset since values will be affected.

Thanks

**Comment by adelra** [comment]
What happened to this problem? I have the same issue
